llvm/utils/lit/tests/Inputs/shtest-cat/cat.txt

```diff
@@ -0,0 +1,83 @@
+# Check cat command with single file.
```

Comment:
```suggestion
## Test cat command with a single file.
```

Reply:
I can make this change in my follow-up patch when I update all the content.